demo-opensuse/README.md

```diff
@@ -0,0 +1,248 @@
+# in-toto openSUSE demo
+In this demo, we will use in-toto to secure a software supply chain for OpenSUSE. Bob is a developer for a project, Carl tests the software before packaging, and Alice oversees the project. So, using in-toto's names for the parties, Alice is the project owner - she creates and signs the software supply chain layout with her private key - and Bob and Carl are project functionaries - they carry out the steps of the software supply chain as defined in the layout.
+
+For the sake of demonstrating in-toto, we will have you run all parts of the software supply chain. That is, you will perform the commands on behalf of Alice, Bob and Carl as well as the client who verifies the final product.
+
+# Setup
+Install docker
+https://docs.docker.com/engine/installation/
+
+Get in-toto
+```
+git clone https://github.com/in-toto/in-toto.git
+```
+
+Build Docker image from Dockerfile
+```
+cd in-toto/demo-opensuse/
+docker pull opensuse
+docker build -t="in-toto-demo-opensuse" .
+```
+Run the docker image
+```
+docker run --privileged -i -t in-toto-demo-opensuse
+```
+
+# Change into the demo directoy and you are ready to start
+```
+cd home/in-toto/
+source in-toto/bin/activate/
```

Comment:
I wonder if we want to have this in the .bashrc or profile so the user doesn't have to do this.